lib/dbDriver/mysql.js

```diff
@@ -8,6 +8,11 @@ class MySQLDriver {
     await knex('project_links')
       .insert({ project_id: project.project_id, label, url });
   }
-}
 
+  async listOutLinks(project) {
```

Comment:
Probably more conventional to use something like `getAllLinksByProject` or something like that. Maybe even use `find` as the verb for continuity.